templates/pages/core/get-started/artik-5-10.md

```diff
@@ -23,3 +25,28 @@ We will walk you through the steps of flashing Ubuntu Core on a Samsung Artik 5
  * Connect the 5V power supply to the board, then use the power button on the board to switch it on
 
 {% include "includes/markdown/get_started_first_boot.md" %}
+
+## Alternative install: Ubuntu Server 16.04 LTS
+
+As an alternative to Ubuntu Core, you can also install Ubuntu Server 16.04 LTS, where you can use your favourite development tools to create and run snaps.
+
+### Ubuntu image
+
+* [ARTIK 5 - Ubuntu Server 16.04 LTS image](http://people.canonical.com/~platform/snappy/artik/archive/artik5-ubuntu-server.img.tar.xz)
+* [ARTIK 10 - Ubuntu Server 16.04 LTS image](http://people.canonical.com/~platform/snappy/artik/archive/artik10-ubuntu-server.img.tar.xz)
+
+Download and copy the image for your board on an SD card by following the [installation media instructions](/core/get-started/installation-medias)
+
+### Installation instructions
+
+Before installing Ubuntu, you need to set your Artik to boot from the SD card, to do so, set the "SW2" switches to `1:on` and `2:on`.
+
+Booting the board from the SD Card will start the Ubuntu installer:
+
+1. Insert the SD card in your Samsung ARTIK
+* Connect the 5V power supply to the board, then use the power button on the board to switch on your Samsung ARTIK
```

Comment:
No need, Markdown interprets it as a numbered list as long as it starts with a 1.